src/util/positioning.ts

```diff
@@ -154,3 +154,9 @@ export function positionElements(
   targetElement.style.top = `${pos.top}px`;
   targetElement.style.left = `${pos.left}px`;
 }
+
+type PlacementSimpleType = 'top' | 'bottom' | 'left' | 'right';
+type PlacementExtendedType = 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right' | 'left-top' | 'left-bottom' |
```

Comment:
What is the added value of splitting `PlacementType` into `PlacementSimpleType` and `PlacementExtendedType`? As far as I can tell we are not using / exporting  `PlacementSimpleType` nor `PlacementExtendedType` anywhere....

Reply:
By defining them differently would help to use the same type for other components e.g.  PlacementSimpleType can be used to set position of datepicker, where PlacementExtendedType is not useful. etc.

Reply:
Why are you saying that `PlacementExtendedType` is not not useful for datepicker? It is actually using one of the "extended types": https://github.com/ng-bootstrap/ng-bootstrap/blob/master/src/datepicker/datepicker-input.ts#L111

Generally speaking, I'm not big fan of adding code that _might_ be used in the future. Let's keep things simple and add more construct as they are needed.